packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -568,3 +569,143 @@ class _TextSelectionHandleOverlayState extends State<_TextSelectionHandleOverlay
     return null;
   }
 }
+
+/// A gesture detector to detect concurrent gesture events for a text field.
+///
+/// Unlike a plain [GestureDetector] where a single gesture wins the gesture
```

Comment:
This widget is unlike a plain [GestureDetector] which only recognizes one tap or double tap in a chain of events.

Reply:
tweaked it